.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "agent-libs"]
+	path = agent-libs
```

Comment:
Since `agent-libs` is one of the dependencies of `probe`, I think we better put the submodule under `kindling/probe`. What do you think?

Reply:
The agent library is a dependency of the probe, and I think it makes sense to do that. I'll put it under the kindling/probe/libs/.